src/main/java/org/myrobotlab/service/InMoovHead.java

```diff
@@ -287,12 +257,6 @@ public void waitTargetPos() {
 
   public void release() {

```

Comment:
should remove the release method if it really should be releaseService.. and if it really should be releaseService.. it should releaseService.